subsys/bluetooth/mesh/Kconfig

```diff
@@ -712,14 +712,15 @@ config BT_MESH_MODEL_EXTENSIONS
 
 config BT_MESH_COMP_PAGE_1
 	bool "Support for Composition Data Page 1"
-	depends on BT_MESH_MODEL_EXTENSIONS
+	default y
 	help
 	  Enable support for Composition Data Page 1.
 
 config BT_MESH_MODEL_EXTENSION_LIST_SIZE
 	int "Model extensions list size"
 	depends on BT_MESH_COMP_PAGE_1
```

Comment:
Should this not rather be dependent on BT_MESH_MODEL_EXTENSIONS, then the default does not need to be 0.

Reply:
No, there are 2 API: `bt_mesh_model_extend` and `bt_mesh_model_correspond`. The first one depends on `CONFIG_BT_MESH_MODEL_EXTENSIONS`, the second one does not. A user can still have the list size > 0 and use only `bt_mesh_model_correspond` API keeping `CONFIG_BT_MESH_MODEL_EXTENSIONS` disabled. `CONFIG_BT_MESH_MODEL_EXTENSIONS` allocates extra RAM in model structs for storing model extension tree, which is not needed when only corresponding relationship is used between models.